go.mod

```
@@ -342,3 +342,7 @@ replace github.com/go-chi/chi/v4 => github.com/go-chi/chi v4.0.0+incompatible
 
 // see https://github.com/ionos-cloud/sdk-go/issues/27
 exclude github.com/ionos-cloud/sdk-go/v6 v6.0.5851
+
+// This replace can be removed once it is patched to v0.3.2 upstream
+// see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/c1a4f58263ec5d1c283827ce97286293aef75631/go.mod#L528
```

```suggestion
// see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/go.mod#L116
```

This is probably the better place to watch as it is the point at which the dependency enters the OTel realm, as a transitive dependency of one of the direct dependencies of the DataDog exporter.

---

oh yes, even a better place to check. thank you